phys/module_sf_noahdrv.F

```diff
@@ -2062,14 +2060,29 @@ SUBROUTINE SOIL_VEG_GEN_PARM( MMINLU, MMINSL)
           IF ( a_string(1:21) .EQ. 'Vegetation Parameters' ) THEN
              CALL wrf_message     ("Expected low and high density residential, and high density industrial information in VEGPARM.TBL")
              CALL wrf_error_fatal ("This could be caused by using an older VEGPARM.TBL file with a newer WRF source code.")
-          ENDIF
-          READ (19,*)LOW_DENSITY_RESIDENTIAL
+        ENDIF
+         READ (19,*)LCZ_1
+          READ (19,*)
+          READ (19,*)LCZ_2
+          READ (19,*)
+          READ (19,*)LCZ_3
+          READ (19,*)
+          READ (19,*)LCZ_4
+          READ (19,*)
+          READ (19,*)LCZ_5
           READ (19,*)
-          READ (19,*)HIGH_DENSITY_RESIDENTIAL
+          READ (19,*)LCZ_6
           READ (19,*)
-          READ (19,*)HIGH_INTENSITY_INDUSTRIAL
+          READ (19,*)LCZ_7
+          READ (19,*)
+          READ (19,*)LCZ_8
+          READ (19,*)
+          READ (19,*)LCZ_9
+          READ (19,*)
+          READ (19,*)LCZ_10
+          READ (19,*)
+          READ (19,*)LCZ_11
```

Comment:
Could you indent these a bit so that we see the logical dependence on the various IF tests.